config-service-change-event-api/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+import com.google.protobuf.gradle.protobuf
+import com.google.protobuf.gradle.protoc
+
+plugins {
+  `java-library`
+  id("com.google.protobuf") version "0.8.17"
+  id("org.hypertrace.publish-plugin")
+}
+
+protobuf {
+  protoc {
+    artifact = "com.google.protobuf:protoc:${libs.versions.protoc.get()}"
+  }
+}
+
+sourceSets {
+  main {
+    java {
+      srcDirs("build/generated/source/proto/main/java")
+    }
+  }
+}
+
+dependencies {
+  api(libs.protobuf.java)
+
+  api(libs.jackson.databind)
```

Comment:
jackson shouldn't be a dependency of the API

Reply:
Removed